opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -364,6 +364,27 @@ def _extract_resources(resource: Resource) -> Dict[str, str]:
     }
 
 
+LABELS_MAPPING = {
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md#attributes
+    "exception.message": "/error/message",
+    "exception.type": "/error/name",
+    "exception.stacktrace": "/stacktrace",
```

Comment:
These are not set in span attributes directly, they will be in an `Event` named "exception".